toolchest_client/tools/api.py

```diff
@@ -470,7 +475,8 @@ def megahit(output_path=None, tool_args="", read_one=None, read_two=None, interl
     return output
 
 
-def python3(script, inputs=[], output_path=None, tool_args="", custom_docker_image_id=None, **kwargs):
+def python3(script, inputs=None, output_path=None, tool_args="", custom_docker_image_id=None,
+            instance_type=InstanceType.COMPUTE_32, volume_size=8, **kwargs):
```

Comment:
We may want to default to a smaller instance class; this would exhaust the free tier pretty quickly